SwEnt-Group8/Street-work-app
# Feat/Use Nominatim API to display better park names

**Overview**

Until now, we displayed only a placeholder value for the name of each park. The objective of this Pull Request is to change those placeholder names with more meaningful names. To do so, we will display the road of each park by using the Nominatim API.

**Features**

- A new repository called `NominatimParkNameRepository` to utilise the Nominatim API
- An update to the UI that now uses this repository to display the road name of each park

**How to Test**

1. Run the Main application.
2. Click on any park.
3. The name of the park should now display the corresponding road name.


**Screenshot**

<img width="232" alt="image" src="https://github.com/user-attachments/assets/5fb4a320-ac48-4aa1-bb00-b3f88500987e">
